GSA/made-in-america
# Dev to stage for Financial Assistance

## Description

udpates stage with dev for changes made to financial assistance page 

## How to test

- visit `https://stage.madeinamerica.gov/financial-assistance/` and ensure the table and page have been updated 

To prepare for code review, please follow the following checks.
